corehq/ex-submodules/casexml/apps/case/views.py

```diff
@@ -32,24 +32,24 @@ def get_display_config(self):
                     [
                         {
                             "expr": "name",
-                            "name": _("Name"),
+                            "name": _("Case Name"),
                             "has_history": True,
                         },
                         {
                             "expr": "opened_on",
-                            "name": _("Opened On"),
+                            "name": _("Date Opened"),
                             "parse_date": True,
                             'is_phone_time': True,
                         },
                         {
                             "expr": "modified_on",
```

Comment:
I think you need to change these expressions to match the table.

Reply:
https://user-images.githubusercontent.com/8924132/104469955-1dbe2280-55df-11eb-997d-b5e37be2f31a.mp4

When I change the `expr` value the data is not retrieved and shows a blank like in the attached video.

Reply:
@proteusvacuum I asked the same to Jenny, she said to ignore expr for now.